com.flashforge.FlashPrint.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
The AppData file lacks some required tags, such as `launchable` tag or `screenshot` tag.
https://github.com/flathub/flathub/wiki/AppData-Guidelines

Reply:
Should I commit the screenshots in the repo ?

The .desktop is pulled from the .deb at install time. Can I link to it in the AppData file ?

Reply:
OK, I think it's done the way you recommended now.